packages.json

```diff
@@ -414,6 +414,7 @@
   "https://github.com/bricklife/JSONRPCKit.git",
   "https://github.com/brightdigit/AssetLib.git",
   "https://github.com/brightdigit/Base32Crockford.git",
+  "https://github.com/brightdigit/GampKit.git",
```

Comment:
Maybe that should be on another PR, so this PR remains clean with the changes on the validation script

Reply:
While it's also showing up in this diff, @leogdion actually submitted this in its own PR. It was merged in #248.